unlock-app/src/components/interface/locks/Manage/elements/WithdrawFundModal.tsx

```diff
@@ -101,23 +111,26 @@ export const WithdrawFundModal = ({
           <Input
             label="Address"
             size="small"
-            {...register('address', {
+            {...register('beneficiary', {
               required: {
                 value: true,
                 message: 'This field is required.',
               },
               minLength: {
                 value: 3,
-                message: 'Address should be 3 characters long at least.',
+                message:
+                  'Beneficiary address should be 3 characters long at least.',
```

Comment:
That's not correct. it should be an ethereum address or ENS. Please use the component that we have that supports that?

Reply:
We don't actually have the component merged, I felt a comment to replace it when it is ready!